manifests/d/DataPLANT/ArcCommander/0.3.3.0/DataPLANT.ArcCommander.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: DataPLANT.ArcCommander
+PackageVersion: 0.3.3.0

```

```suggestion
PackageVersion: 1.0.0.0
```
This must match Control Panel to prevent upgrade loops due to version mismatch.